extensions/wikia/PowerUser/PowerUser.class.php

```diff
@@ -69,6 +71,22 @@ function __construct( \User $oUser ) {
 	}
 
 	/**
+	 * @var UserPermissions
+	 */
+	private $permissionsService;
+
+	/**
+	 * @return UserPermissions
+	 */
+	private function userPermissions() {
```

Comment:
this method and the private field is repeated unchanged in many classes, good candidate for `trait` ?